src/app/DTT/address/get_address/index.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useRef, useState } from 'react'
+import { Input } from '@chakra-ui/react'
+import { isEmpty, map } from 'ramda'
+import initialiseMap from './helpers/initialise-map'
+
+const GetAddress = ({ onSendAnswer, data }) => {
+  const inputRef = useRef(null)
+  const [searchAddress, setSearchAddress] = useState('')
+  const [allOptions, setAllOptions] = useState([])
+  const [showSuggestiongs, setShowSuggestions] = useState(true)
+  const handleOptionClick = description => {
+    setSearchAddress(description)
+    setShowSuggestions(false)
+    onSendAnswer(description)
+  }
+
+  useEffect(() => {
+    initialiseMap(inputRef, searchAddress, allOptions, setAllOptions)
```

Comment:
are we initalising the map every time searchAddress or allOptions changes? should initialiseMap only be called once? Can you explain why this happens